src/firebase/firebase-admin.service.ts

```diff
@@ -0,0 +1,19 @@
+import * as admin from 'firebase-admin';
+import { Injectable } from '@nestjs/common';
+
+@Injectable()
+export class FirebaseAdminService {
+  constructor() {
+    const serviceAccount = require('../../firebase/firebase-adminsdk.json'); // 수정 필요
```

Comment:
리뷰 요구사항 부분이네요.
먼저 저 json 파일을 로컬에 저장한 후에 터미널 명령어로 
( 이런식이라네요  `base64 firebase-adminsdk.json | tr -d '\n'`)
를 입력하면 한 줄의 인코딩 문자열이 나오는데

이걸 그대로 복사해서 env에 환경변수로 등록 한 후 ( 예를 들어 `FIREBASE_ADMIN_SDK` = ~ 키내용 ~ )

```suggestion
    const serviceAccount = JSON.parse(
      Buffer.from(configService.get<string>('FIREBASE_ADMIN_SDK'), 'base64').toString('utf8'),
);
```
이런 식으로 환경변수를 불러오고 파싱 명령어로 원래의 json 처럼 사용하면 안전하다네요~

json 그대로 env에 등록해도 되지만 특수문자 때문에 제대로 입력 안 될 경우가 많대요

Reply:
자세한 설명 덕분에 바로 작업 가능했어요~!! 
수정했습니다 ㅎ